pkg/antctl/raw/update/clickhouse/update_clickhouse.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouse
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	flowaggregatorconfig "antrea.io/antrea/pkg/config/flowaggregator"
+)
+
+// Command is the support bundle command implementation.
+var Command *cobra.Command
+
+var option = &struct {
+	enable         string
+	database       string
+	databaseURL    string
+	debug          string
+	compress       string
+	commitInterval string
+	ttl            string
+}{}
+
+var example = strings.Trim(`
+  Enable clickhouse
+  $ antctl update clickhouse --enable true/false
+  Update database
+  $ antctl update clickhouse --database name
+  Update databaseURL
+  $ antctl update clickhouse -d Http://xxxxx
+  Update debug
+  $ antctl update clickhouse --debug true
+  Update compress
+  $ antctl update clickhouse --compress true
+  Update commitInterval
+  $ antctl update clickhouse -i 10s
+  Update ttl
+  $ antctl update clickhouse --ttl 100
```

Comment:
can we have a more uniform scheme to update configuration fields? I think I suggested using the "YAML path" in the past, e.g. `antctl set clickHouse.enable=true`. Would there be any issue with that approach?

Reply:
I have one question. When I use "YAML path", e.g. `antctl set clickHouse.enable=true clickhouse.ttl=100`, should I treat "clickHouse.enable=true clickhouse.ttl=100" as a string and parse it, or is there any better method I can use to approach this one?

Reply:
I think you don't want to try to do anything too fancy here (e.g. using reflection). A pretty straightforward solution would be to have a map between path and config mutator function. Something like this:
```go
type FlowAggregatorConfigMutator func(c *config.FlowAggregatorConfig, value string) error

func setBoolOrFail(b *bool, value string) error {
    if value != "true" && value != "false" {
        return fmt.Errorf(...)
    }
    *b = (value == "true")
    return nil
}

mutators := map[string]FlowAggregatorConfigMutator{
    {"clickHouse.enable": func(c *config.FlowAggregatorConfig, value string) error { return setBoolOrFail(&c.ClickHouse.Enable, value) }},
    // ...
}
```

If you think of something less verbose and as readable, go for it.